credential/cargo-credential/Cargo.toml

```diff
@@ -7,6 +7,8 @@ repository = "https://github.com/rust-lang/cargo"
 description = "A library to assist writing Cargo credential helpers."
 
 [dependencies]
+anyhow.workspace = true
```

Comment:
`anyhow` could be made an optional dependency with a feature to enable it. It's here to enable a convenience conversion.

Reply:
Not the biggest fan of things like `anyhow` being in library APIs **but** the place we care most about compatibility is the wire protocol and `anyhow` doesn't matter there so ... meh.